### PR TITLE
Sprint 9 - update answered questions on re-record

### PR DESCRIPTION
**Pull request by lynnebai**
- created delete route '/questions' that takes question ID in body and removes the question from user's Answered list
- call the route when user confirms re-record of an already-answered question on Questions screen

### CODE REVIEW

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ interface Props {
 export default function QuestionsScreen(props: Props) {
   const [selected, setSelected] = useState<number | null>(null)
   const [questions, setQuestions] = useState<Array<Question>>([])
+  let questionToRemove = 0
```
Comment (apetranik) [inline src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:29]: make this a state variable

Comment (lynnebai) [reply src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:29]: when I try to use setState for it, its read as undefined when passed in the body of the delete route, I don't know how to make it work

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx [hunk 2]
```diff
@@ -41,6 +42,26 @@ export default function QuestionsScreen(props: Props) {
       })
   }, [])
 
+  async function removeQuestion(){
```
Comment (apetranik) [inline src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:45]: take in `id: number` as an argument

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx [hunk 3]
```diff
@@ -56,12 +77,14 @@ export default function QuestionsScreen(props: Props) {
...
                     {text: 'Re-record Answer', 
                       onPress: () => {
+                        questionToRemove = item.ID
                         Alert.alert(
                           'Are you sure you want to re-record your clip?',
                           'You\'ll lose your old clip',
                           [
                             {text: 'Re-record',
                               onPress: () => {
+                                removeQuestion()
```
Comment (apetranik) [inline src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:87]: call removeQuestion(item.ID)

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx [hunk 4]
```diff
@@ -56,12 +77,14 @@ export default function QuestionsScreen(props: Props) {
                     {text: 'View Answer'},
                     {text: 'Re-record Answer', 
                       onPress: () => {
+                        questionToRemove = item.ID
```
Comment (apetranik) [inline src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:80]: get rid of and refer to below comment

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx (hunk 1 above)
Comment (apetranik) [reply src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:29]: just get rid of this altogether and refer to below suggestion

File: src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx [hunk 5]
```diff
@@ -41,6 +42,26 @@ export default function QuestionsScreen(props: Props) {
       })
   }, [])
```
Comment (apetranik) [inline src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx:43]: remove the `, []` from this. not your fault but it's causing some issues w/ reloading the page

**Pull request merged**